Interop/PInvoke.cs

```diff
@@ -38,6 +38,9 @@ internal class PInvoke
 		internal static extern IntPtr tidyReleaseDate();
```

Comment:
Are we still using this? (And clean up the new one, too, please if it stops being needed.)